### PR TITLE
f-checkout@v0.173.3 Improve logging

### PR DESCRIPTION
**Pull request by BanksySan**
Added extra items to the logged data when a user attempts to submit an invalid checkout form.

### CODE REVIEW

**Review by SachaZvetelman — DISMISSED** [review]

File: packages/components/organisms/f-checkout/src/components/Checkout.vue [hunk 1]
```diff
@@ -988,9 +992,18 @@ export default {
...
+            const expandedData = {
+                ...this.eventData,
+                enteredPostcode: this.address?.postcode,
+                location: this.$cookies.get('je-location'),
+                locationUk: this.$cookies.get('je-location-uk'),
+                changedFields: this.changedFields,
+                isPostcodeChanged: this.changedFields.includes('addressPostcode')
+            };
+
             this.logInvoker({
```
Comment (charliearlie) [inline packages/components/organisms/f-checkout/src/components/Checkout.vue:1004]: I believe there's a test for this which may need updating. Including mocking cookies.

File: packages/components/organisms/f-checkout/src/components/Checkout.vue [hunk 2]
```diff
@@ -988,9 +992,18 @@ export default {
                 error: ANALYTICS_ERROR_CODE_INVALID_MODEL_STATE
             });
```
Comment (charliearlie) [inline packages/components/organisms/f-checkout/src/components/Checkout.vue:993]: Should we remove the invalid model state log as well @SachaZvetelman ?

File: packages/components/organisms/f-checkout/src/components/Checkout.vue (hunk 1 above)
Comment (charliearlie) [reply packages/components/organisms/f-checkout/src/components/Checkout.vue:1004]: `checkout.test.js` from line 3452 🙂

File: packages/components/organisms/f-checkout/src/components/Checkout.vue (hunk 2 above)
Comment (SachaZvetelman) [reply packages/components/organisms/f-checkout/src/components/Checkout.vue:993]: That's a really good question. I'll need to confirm with Jack. Maybe leave it for the time being and we can remove it later so this PR doesn't get delayed.

Comment (charliearlie) [reply packages/components/organisms/f-checkout/src/components/Checkout.vue:993]: Yep, I just mentioned it as it was visible here! I'll take a note to remember to follow up.

Comment (BanksySan) [reply packages/components/organisms/f-checkout/src/components/Checkout.vue:993]: I'd rather keep the PR focused, I'm quite happy to create a new one though is there's tidy-up outside of this.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "3.53.1",
```
Comment (SachaZvetelman) [inline package.json:3]: Do we need to bump this now when we modify one of our components? I didn't know.

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "3.53.2",
```
Comment (SachaZvetelman) [inline package.json]: Do we need to bump this now when we modify one of our components? I didn't know.

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v3.53.2
+------------------------------
+*August 24, 2021*
+
+### Added
+- f-checkout logging items to aid debugging conversion drop.
+
```
Comment (SachaZvetelman) [inline CHANGELOG.md]: Same here.

File: package.json (hunk 4 above)
Comment (BanksySan) [reply package.json]: I assumed do.  It's probably sensible to increment the minor version as it's a different codebase but not a change in functionality.

Comment (SachaZvetelman) [reply package.json]: We only bump the version and update the CHANGELOG of the root when we modify something at the root level. If we only modify components, we bump and update the CHANGELOGs of said components. So in this case I think we need to revert this change.

**Review by charliearlie — APPROVED** [review]

**Review by tommcclean — APPROVED** [review]

**Pull request merged**